test/sslapitest.c

```diff
@@ -352,7 +376,11 @@ static int test_keylog_no_master_key(void)
 
     if (!TEST_true(create_ssl_ctx_pair(TLS_server_method(), TLS_client_method(),
                                        TLS1_VERSION, TLS_MAX_VERSION,
-                                       &sctx, &cctx, cert, privkey)))
+                                       &sctx, &cctx, cert, privkey))
+        || !TEST_true(SSL_CTX_set_max_early_data(sctx,
+                                                 SSL3_RT_MAX_PLAIN_LENGTH))
+        || !TEST_true(SSL_CTX_set_max_early_data(cctx,
+                                                 SSL3_RT_MAX_PLAIN_LENGTH)))
```

Comment:
It isn't necessary to set max early data on the client side. This is a server side setting.

Reply:
If that is the case, then it should be removed from `setupearly_data_test` as well?

Reply:
Yes it should. Feel free to do that here, or leave it to another PR.